TOPdeskPS/functions/Change Management/Set-TdChange.ps1

```diff
@@ -0,0 +1,57 @@
+﻿function Set-TdChange {
+    <#
+.SYNOPSIS
+    Sort of sets a change, this is poorly supported by TOPdesk :/
+.DESCRIPTION
+    does most of the change setting, poorly supported by TOPdesk, read the .NOTES
+.PARAMETER ChangeId
+    Id of the Change Request to be altered
+.PARAMETER BodyInput
+    formatted input to match the example value linked to in the .NOTES
+.PARAMETER Confirm
+    If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
+.PARAMETER WhatIf
+    If this switch is enabled, no actions are performed but informational messages will be displayed that explain what would happen if the command were to run.
+.EXAMPLE
+    PS C:\> Set-TdChange -ChangeId 'C1807-627' -BodyInput 'body text'
+    sets the body of specified change. this api is poorly supported by TOPdesk
+
+.NOTES
+    https://developers.topdesk.com/explorer/?page=change#/Working%20as%20an%20operator/patch_operatorChanges__identifier_
```

Comment:
use .LINK instead of .NOTES
 .LINK
The name of a related topic. The value appears on the line below the ".LINK" keyword and must be preceded by a comment symbol # or included in the comment block.

Repeat the ".LINK" keyword for each related topic.

Reply:
fixed